main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedExpirationDateValidator.java

```diff
@@ -73,13 +74,32 @@ public void validate(GtfsFeedInfo entity, NoticeContainer noticeContainer) {
     }
   }
 
-  static class FeedExpirationDateNotice extends ValidationNotice {
+  static class FeedExpirationDateNotice7Days extends ValidationNotice {
     private final long csvRowNumber;
     private final GtfsDate currentDate;
     private final GtfsDate feedEndDate;
     private final GtfsDate suggestedExpirationDate;
 
-    FeedExpirationDateNotice(
+    FeedExpirationDateNotice7Days(
+        long csvRowNumber,
+        GtfsDate currentDate,
+        GtfsDate feedEndDate,
+        GtfsDate suggestedExpirationDate) {
+      super(SeverityLevel.WARNING);
+      this.csvRowNumber = csvRowNumber;
+      this.currentDate = currentDate;
+      this.feedEndDate = feedEndDate;
+      this.suggestedExpirationDate = suggestedExpirationDate;
```

Comment:
Maybe we don't have to, but if you look at the original `FeedExpirationDateNotice`, the field was already included. If we decide to remove it, maybe it would open another PR to make the refactor.